text/0000-annotations.md

```diff
@@ -0,0 +1,78 @@
+# Meta
+[meta]: #meta
+- Name: OCI Image annotations
+- Start Date: 2021-12-03
+- Author(s): [@samj1912](https://github.com/samj1912)
+- Status: Draft <!-- Acceptable values: Draft, Approved, On Hold, Superseded -->
+- RFC Pull Request: (leave blank)
+- CNB Pull Request: (leave blank)
+- CNB Issue: (leave blank)
+- Supersedes: (put "N/A" unless this replaces an existing RFC, then link to that RFC)
+
+# Summary
+[summary]: #summary
+
+Add OCI recommended image annotations to output images.
+
+# Definitions
+[definitions]: #definitions
+
+N/A
+
+# Motivation
+[motivation]: #motivation
+
+Certain OCI image scanning and debugging tools take into account standard annotations on the images as defined by [OCI Spec - Pre-defined Annotation keys](https://github.com/opencontainers/image-spec/blob/main/annotations.md#pre-defined-annotation-keys). Buildpacks already records some of this information in the `project.toml` and other labels defined at https://github.com/buildpacks/spec/blob/main/platform.md#labels. We should also preserve this information in the output image in appropriate annotations as recommended by the OCI spec.
+
+# What it is
+[what-it-is]: #what-it-is
+
+Adds OCI recommended annotations to the output image.
+
+# How it Works
+[how-it-works]: #how-it-works
+
+Optionally adds the following annotations where applicable when lifecycle is used to publish images directly to the registry - 
+
+- `org.opencontainers.image.base.name`: Run image tag name. Currently stored in `io.buildpacks.lifecycle.metadata.run-image.reference`
+- `org.opencontainers.image.base.digest`: Run image digest. Currently stored in `io.buildpacks.lifecycle.metadata.run-image.reference`
```

Comment:
Does the `run-image.reference` data include both the mutable tag reference and the immutable digest?

Reply:
Based on the [spec](https://github.com/buildpacks/spec/blob/main/platform.md#iobuildpackslifecyclemetadata-json) this would be the immutable digest:

> A digest reference to a manifest stored in an OCI image registry